app/src/main/res/layout/item_breed.xml

```diff
@@ -15,22 +15,21 @@
             android:layout_width="wrap_content"
             android:layout_height="32dp"
             android:gravity="center_vertical"
+            android:text="@string/breed"
```

Comment:
I thought about this, but idk if in the case of failing to initialize blank is better than a default like this. I could see either way

Reply:
Yeah. Not a huge deal since it's just sample code anyway. Don't need to dig to hard on Android best-practices when most people's focus will be on KMP and iOS.